### ISSUE TITLE
Move over backups to smithers

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/smithers#1]
Will also include peta snapshot of bayes for things that won't change.

**Comment by jonathanolson** [comment]
Manual snapshots to be made:

- /data/share/phet/statistics-data/raw/ (should be sufficient)

**Comment by jonathanolson** [comment]
Executing on peta-snapshot screen on bayes:
```sh
scp -r /data/share/phet/statistics-data/raw phet-admin@dtn-data.rc.int.colorado.edu:/work/phet/bayes/raw-snapshot-2017-07-10
```

**Comment by jonathanolson** [comment]
All scripts are moved over.

phet-server scripts completed first and are fully working.

Waiting on both bayes tests.

**Comment by jonathanolson** [comment]
Bayes weekly looks like it succeeded, running twice due to the incremental backup feature.

**Comment by jonathanolson** [comment]
@oliver-phet, are you familiar with what our disk quotas are for the peta library, and/or what we might be paying? I may be starting to hit the quotas.

It is probably possible to work around this, but it would help to know the limit.

**Comment by ariel-phet** [comment]
@jonathanolson 

https://www.rc.colorado.edu/resources/storage/petalibrary/service

> While we gauge demand for the different options, new customers will initially be limited to 75 TB in Active Storage or 100 TB in Archive Storage.  

Are you really nearing 100 TB? That seems a little "log obsessive" to me if it is the case

**Comment by jonathanolson** [comment]
> Are you really nearing 100 TB? That seems a little "log obsessive" to me if it is the case

I don't think so, but if the "incremental" part of the backup broke and made copies of everything daily/weekly/monthly, it's not out of the question.

I've been running a command to determine the size of our content, but it's taking a while (which it may need to do, as the backups should have lots of hard links to the same file).

**Comment by jonathanolson** [comment]
Interrupted by service issues (https://rc.colorado.edu/node/1036).

Can't log in to research computing right now, and my previous SSH session had "Write failed: Broken pipe".

**Comment by jonathanolson** [comment]
Just kicked off another `du -sh` to determine the actual size under /work/phet.

**Comment by jonathanolson** [comment]
Had a D/C before it finished, now checking subsets of our content so they'll hopefully finish.

**Comment by jonathanolson** [comment]
Have three things running on separate screen instances. Disk usage without those:
```
apache-log-backups 104GB
bayes 224GB
figaro 116GB
figaro.backup.... 314GB
old ------
phet-cache1 48GB
phet-cache2 48GB
phet-server --()
	htdocs------
	offline-statistics 19GB
	phet-io ------
	website-database 19GB
simian 25GB
tomcat-logs-backup 0.8GB
varnish-logs-backup 26GB

total: 944GB so far
```

**Comment by jonathanolson** [comment]
It look like `du -sh` is properly picking up deduplication in most places it is expected. For instance, running it with multiple files:

```
login02:/work/phet/phet-server/htdocs # du -sh 2017-03-*
65G	2017-03-01-1488355201
265M	2017-03-02-1488441601
156M	2017-03-03-1488528001
158M	2017-03-04-1488614401
```
shows subsequent snapshots taking up "less" space, but then if we inspect a single one (that reported as 265M above):
```
login02:/work/phet/phet-server/htdocs # du -sh 2017-03-02-1488441601
65G	2017-03-02-1488441601
```
it shows properly as the full size.

Even if there was no duplication, htdocs would be ~30TB and phet-io would be <0.5TB, so as a high-end estimate, if everything is broken, we would not be weighing in at more than 45TB of data.
